.github/workflows/allure-report.yml

```
@@ -93,31 +93,31 @@ jobs:
 
           # categories.json — failure type classification
           cat > allure-results/categories.json <<'CATEOF'
-          [
-            {"name": "Test Defects", "matchedStatuses": ["failed"]},
-            {"name": "Infrastructure Issues", "matchedStatuses": ["broken"]},
-            {"name": "Skipped / Disabled", "matchedStatuses": ["skipped"]}
-          ]
-          CATEOF
+[
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)